src/createNumbersGenerator.js

```diff
@@ -26,7 +26,22 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  let count = 0;
+
+  return () => {
+    if (!numbers) {
+      return;
+    }
+    for (let i = count; i < numbers.length; i++) {
+      count++;
+      if (!min && !max) {
+        return numbers[i];
+      }
+      if (numbers[i] >= min && numbers[i] <= max) {
```

Comment:
good, but if we will receive only `max` or `min`?

you can solve this problem with another check or add default values to min and max